apps/src/util/experiments.js

```diff
@@ -15,6 +15,8 @@ const STORAGE_KEY = 'experimentsList';
 const GA_EVENT = 'experiments';
 const EXPERIMENT_LIFESPAN_HOURS = 12;
 
+experiments.SHARE_SETTING = 'shareSetting';
```

Comment:
In general, we haven't been doing this sort of thing unless the experiment is used in lots of places. That said, it might be a good practice to start doing more of this sort of thing, if for no other reason than to have some sort of documentation of which experiments we currently have available. I don't feel too strongly either way.

Reply:
Yeah, I noticed it wasn't in here, but it felt like a good idea to do so we could have a source of truth.